lms/djangoapps/grades/management/commands/reset_grades.py

```diff
@@ -89,8 +95,16 @@ def handle(self, *args, **options):
 
         operation = self._query_grades if run_mode == 'dry_run' else self._delete_grades
 
-        operation(PersistentSubsectionGrade, course_keys, modified_start, modified_end)
-        operation(PersistentCourseGrade, course_keys, modified_start, modified_end)
+        db_table = options.get('db_table')
+
+        if db_table not in {'subsection', 'course', None}:
+            raise CommandError('Invalid value for db_table. Valid options are "subsection" or "course" only.')
```

Comment:
Please add test coverage for this exception.  Similar to the `test_invalid_key` method.